### ISSUE TITLE
Fix  Combobox aria-activedescendant value

### ISSUE THREAD

**Issue by simonssspirit** [telerik/kendo-react#1216]
Fix  Combobox aria-activedescendant value.

Currently, the value of the input is not 100% the same as the one in the li elements. It is missing the option value:

The id of the li:

![image](https://user-images.githubusercontent.com/11087921/153346539-9354433c-76b6-4394-86e2-d70e6e3a926a.png)

The aria-activedescendant value of the input:

![image](https://user-images.githubusercontent.com/11087921/153346602-a20d9afb-d90d-484e-a493-0c1ce74fc6e0.png)

As a result, the value is not read by some of the screenreaders.

**Comment by InaGlushkova** [comment]
Introduced with dev version [v5.4.0](https://www.telerik.com/kendo-react-ui-develop/components/changelogs/ui-for-react/#861151b6-4033-51db-8a3f-1e84d4991fd3)

More information about obtaining the development builds - 
https://www.telerik.com/kendo-react-ui/components/installation/development-builds/
